### ISSUE TITLE
[Q] Is it possible that: On the same run, and on under the same metric to have multiple graphs?

### ISSUE THREAD

**Issue by miguelvalente** [wandb/client#2675]
The questions pretty much sums up the issue.

In my training script I train a classifier multiple times. I would like to be able to make new graphs under the same metric every time I train a classifier from scratch. [Example of my Issue](https://wandb.ai/mvalente/Zero_Flow_CUB/reports/Shared-panel-21-09-16-18-09-00--VmlldzoxMDI5Mzky?highlightShare) .

[What I would like to happen](https://wandb.ai/mvalente/toy_data_zf/reports/Shared-panel-21-09-16-18-09-04--VmlldzoxMDI5Mzk2?highlightShare) Some sort of slider or anything that achieves the same functionality to have the graphs separated.

Thx in advance!

**Comment by cvphelps** [comment]
@miguelvalente thanks for writing in to ask about this! It sounds like you want to log a line plot at every step of training and then scrub across the run to see how it changes.

One approach I might recommend here is to call wandb.init() every time you train a new classifier. Could you describe why you're putting all these charts into a single run instead of having one run for each instance of training?

If you're trying to associate a bunch of individual trained model versions together in one larger experiment, you might benefit from the Grouping Feature: https://docs.wandb.ai/guides/track/advanced/grouping

**Comment by miguelvalente** [comment]
@cvphelps Hey thanks for answering,

I'm training a generative NN and running a classifier on the generated results every _N_ epochs, so its the same instance of a generator NN but different classifiers. The classifiers training is part of the training loop of the generator. Maybe a feature to implement!

I can always just change the graph. The simplest way would be to change the line graph to points to at least remove the lines that appear in between runs of the classifier. It would make the graphs cleaner but the x_scale would still be off and with a lot of empty space in between. 

Thx for your time!
